pages/_app.js

```diff
@@ -2,17 +2,55 @@ import GlobalStyle from "../styles";
 import { initialEmotionEntries } from "@/lib/initialEmotionEntries";
 import { v4 as uuidv4 } from "uuid";
 import useLocalStorageState from "use-local-storage-state";
+import ToastMessage from "@/components/ToastMessage";
+import { useState, useEffect } from "react";
 
```

Comment:
Consider extracting magic numbers into constants at the top of the file:

e.g. 

``` js
const TOAST_LIMIT = 3;
const TOAST_DURATION = 3000;
const ANIMATION_DURATION = 300;
```

This makes the timing and limits more self-documenting and easier to modify.